### ISSUE TITLE
Using html

### ISSUE THREAD

**Issue by mifi** [rhysd/electron-about-window#2]
Nice module!

Would be nice to have the option to put HTML inside the attributes (like description)

**Comment by rhysd** [comment]
Could you show me some code snippet example? Let's discuss based on code.

**Comment by mifi** [comment]
Example:
```
openAboutWindow({
  icon_path: join(__dirname, 'icon.png'),
  description: '<b>Some bold test</b><a href="myawesomeapp.com">See more here</a>.'
                            })
```
Currently it just renders the html markup as text, because in the code you're using innerText (https://github.com/rhysd/electron-about-window/blob/master/src/renderer.ts#L30)
Could maybe use innerHTML instead.

**Comment by rhysd** [comment]
Thanks. I think implementation is very simple as you said. Let me think there is no risk about XSS because of setting inner HTML.

**Comment by rhysd** [comment]
I noticed that your example doesn't work. `<a>` tag navigates to your project page in the 'About Me' window. The window is not protected with sandbox. So we should not navigate to any external page in it.

Regarding to your use case, is it also acceptable to add 'See more' link to description?

**Comment by rhysd** [comment]
I added a link to application's homepage to application's name instead of adding 'see more' to description because longer description may break the layout. Change is 1e0f5e6.

Does it meet your usecase?

**Comment by mifi** [comment]
Oh I see. That sounds like a nice addition too (the link.)

Actually the link wasn't a particular use case, more an example. It was more that it would be nice to be able to add custom HTML block, if someone needs to write some more text with formatting in their about page.

**Comment by rhysd** [comment]
OK, I think we can use `will-navigate` event to prevent navigation and open the URL with external browser. I'll try to implement

**Comment by rhysd** [comment]
```js
openAboutWindow({
                                icon_path: join(__dirname, 'icon.png'),
                                copyright: 'Copyright (c) 2015 rhysd',
                                description: 'Hello, world <a href="https://github.com" class="link">(see more)</a>'
})
```

now opens a window like below:

![2016-11-07 2 28 12](https://cloud.githubusercontent.com/assets/823277/20039986/0b78774a-a492-11e6-914c-b8c7e89dd95f.png)

I added `will-navigate` hook and `new-window` hook.  So `<a>` tag will open the link with external browser.

I published v1.5.0.

**Comment by mifi** [comment]
Great! thanks! However I think changing from innerText to innerHtml is a breaking change, so we should either bump version to 2.0.0 or introduce a flag in order to use innerHtml instead of innerText.
Maybe introduce a property called "useHtml" ?

**Comment by rhysd** [comment]
Hmm... I think adding new property is better because innerHTML is dangerous.  In the case, current behavior is not broken. I'll unpublish 1.5.0 and add `useInnerHTML` option in 1.6.0.

**Comment by mifi** [comment]
You can have useInnerHtml property in 1.5 as well, as long as it defaults to innerText, because then it is backwards compatible :)

**Comment by rhysd** [comment]
Following semantic versioning, we need to up minor version because it's a new feature without breaking change. Anyway, I already published v1.6.0 👍
